apps/ui/src/components/pages/application/SectionMenu.tsx

```diff
@@ -91,7 +93,7 @@ const SectionMenu = ({ currentSection, isEditMode, appId, revisions, appState }:
 			selectedKeys={[currentSection]}
 			mode="inline"
 			items={
-				!isLoading
+				!isLoading && role
```

Comment:
Isn't `ANONYMOUS` a role, what does this check accomplish? Is it potentially undefined? Do we need an `isLoading` for user context?